packages/e2e-tests/specs/editor/blocks/quote.test.js

```diff
@@ -89,7 +91,7 @@ describe( 'Quote', () => {
 			await page.keyboard.type( 'two' );
 			await page.keyboard.press( 'ArrowRight' );
 			await page.keyboard.type( 'cite' );
-			await transformBlockTo( 'Paragraph' );
+			await transformBlockTo( 'Unwrap' );
```

Comment:
This test is failing because after `ArrowRight` you can't type a `cite`. See [keyboard interactions](https://github.com/WordPress/gutenberg/pull/25892). We need to either fix this keyboard interaction or adapt the test.